packages/swapper/src/swappers/JupiterSwapper/swapperApi/getTradeRate.ts

```diff
@@ -198,11 +198,11 @@ export const getTradeRate = async (
 
   const tradeRate: TradeRate = {
     id: uuid(),
+    quoteOrRate: 'rate',
     rate: inputOutputRate,
     receiveAddress: undefined,
```

Comment:
We are not forced to explicitly set this to undefined now then, as it can be defined now